instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from './CommentForm';
+import Comment from './Comment';
+
+// import PostContainer from './PostContainer/PostContainer';
+
+class CommentSection extends React.Component {
+    constructor(props){
+        super(props)
+        console.log(props.post.comments)
+        this.state = {
+            comments: props.post.comments
+        }
+    }
+
+addNewComment = (event, indexNum, commentText) => {
+    const copiedComments = this.state.comments
+    const arrayWithNewComment = copiedComments.push(commentText)
+    this.setState({comments: arrayWithNewComment})
+}
+
+render() {
+    return (
+
+        <div>
+        <CommentForm postId={this.props.id} addNewComment={this.addNewComment} />
+
+        </div>
+    )
+}
+
+
+CommentSection.propTypes = {
```

Comment:
This should not be nested inside of your class. You can put this whole section above where you export the CommentSection component at the bottom of this file